simple.js

```diff
@@ -1071,6 +1078,48 @@ define(function(require, exports, module) {
             });
         }
 
+
+        /**
+         * Spawns adminer and opens it on port 8082 in a new browser tab
+         */
+        function startAdminer() {
+
+            // open new browser tab
+            var tab = window.open("", "_blank");
+            if (!tab)
+                return;
+
+            tab.document.write(
+                'Starting adminer...<br>' +
+                'Please wait! This page will reload automatically.'
+            );
+
+            proc.spawn("adminer", { args: [ "--url-only" ] }, function(err, process) {
```

Comment:
Is it C9 style to have spaces inside arrays?

Reply:
I thought it was, but maybe I saw that somewhere else.